### PR TITLE
:robot: Triggering CI on branch 'release-next' after synching to upstream/master

### PR DESCRIPTION
**Pull request by chmouel**

### CODE REVIEW

**Comment by chmouel** [comment]
We need to get https://github.com/openshift/tektoncd-pipeline/pull/253 first 

/cc @vdemeester

File: openshift/release/tektoncd-pipeline-nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,954 @@
...
+    spec:
+      serviceAccountName: tekton-pipelines-controller
+      containers:
+      - name: tekton-pipelines-controller
+        image:  quay.io/openshift-pipeline/tektoncd-pipeline-controller:nightly
+        args: [
+          "-kubeconfig-writer-image", "quay.io/openshift-pipeline/tektoncd-pipeline-kubeconfigwriter:nightly",
+          "-creds-image", "quay.io/openshift-pipeline/tektoncd-pipeline-creds-init:nightly",
+          "-git-image", "quay.io/openshift-pipeline/tektoncd-pipeline-git-init:nightly",
+          "-nop-image", "quay.io/openshift-pipeline/tektoncd-pipeline-nop:nightly",
```
Comment (chmouel) [inline openshift/release/tektoncd-pipeline-nightly.yaml]: Should be better 🤗

**Comment by alanfx** [comment]
OCF Webhook is merging this PR

**Pull request merged**